astropy/wcs/wcs.py

```diff
@@ -129,6 +130,11 @@
 # Additional relax bit flags
 WCSHDO_SIP = 0x10000
 
+# Regular expression defining SIP keyword
+# It matches keyword that starts with A, B, AP or B, followed by an underscore then
+# a number in range of 0-19, followed by an underscore and another number
+# in range of 0-19. Keyword optionally ends with a capital letter.
```

Comment:
@plim Alternate WCS can be denoted with any of the letters in the English alphabet - hence the limit of 26 alternate WCS.  Feel free to suggest anything better as a substitute of `<L>` . It has the same meaning as the keyword `key` in the WCS constructor.